release-notes/8.0/8.0.0/api-diff/Microsoft.NETCore.App/8.0.0_System.Runtime.CompilerServices.md

```diff
@@ -0,0 +1,123 @@
+# System.Runtime.CompilerServices
```

Comment:
System.Runtime.CompilerServices LGTM

Namely new attributes to support language features and remove the need for the compiler to emit internal copies in every assembly. But also some changes to use `ref readonly` where applicable

CC. @dotnet/area-system-runtime-compilerservices